tests/src/Context/DrupalContext.php

```diff
@@ -107,36 +100,6 @@ public function resetNodes() {
   }
 
   /**
-   * Remember the list of node type.
-   *
-   * @BeforeScenario @reset-node-types
```

Comment:
I talked to remove this tag on the page_layout feature, why do you prefer this way?